README.md

```diff
@@ -276,6 +276,30 @@ func HandleRequest(ctx context.Context, name string) (string, error) {
 }
 ```
 
+`aws-xray-sdk-go` doesn't currently support streaming grpc call.
+
+**grpc client**
+
+```go
+conn, err := grpc.Dial(
+    serverAddr,
+    grpc.WithInsecure(),
+    grpc.WithUnaryInterceptor(grpcmiddleware.ChainUnaryClient(
+        xray.UnaryClientInterceptor(serverAddr),
+    )),
+)
+```
+
+**grpc server**
+
+```go
+grpcServer := grpc.NewServer(
+    grpcmiddleware.WithUnaryServerChain(
```

Comment:
We should probably include below imports here to give clear idea about `grpcmiddleware` package

```
grpc_middleware "github.com/grpc-ecosystem/go-grpc-middleware"
```

Reply:
`grpc` had added similar chaining methods so we don't need to use `grpc_middleware` now.

To apply the interceptors, we can use [`grpc.UnaryInterceptor`](https://pkg.go.dev/google.golang.org/grpc#UnaryInterceptor) r [`grpc.WithUnaryInterceptor`](https://pkg.go.dev/google.golang.org/grpc#WithUnaryInterceptor) directly. So I simplify the examples a bit and add comments to show how to apply multiple interceptors.